lib/cli/errors.js

```diff
@@ -18,6 +18,13 @@ exports.handleErrors = function(error) {
     return 1;
 };
 
+exports.handleUncaughtExceptions = function() {
+    process.on('uncaughtException', function(err) {
+        console.error(err);
```

Comment:
I would use `process.exit(handleErrors(error))` to get an output and exit code consistent with the rest of the error reports.